@xen-orchestra/fs/src/local.js

```diff
@@ -92,7 +92,10 @@ export default class LocalHandler extends RemoteHandlerAbstract {
   }
 
   async _readFile(file, options) {
-    return fs.readFile(this._getFilePath(file), options)
+    return retry(() => fs.readFile(this._getFilePath(file), options), {
+      tries: 2,
```

Comment:
It would be better as an option, see `mountDir` or `useSudo` there: https://github.com/vatesfr/xen-orchestra/blob/11801f306c1f88c257d3293c7c8d483068b66da7/%40xen-orchestra/fs/src/_mount.js#L12